docs/README.md

```diff
@@ -0,0 +1,194 @@
+# Documentation for Nix-Doom-Emacs
+
+Nix-Doom-Emacs (also commonly referred to as NDE in chatrooms) is a project with lots of moving pieces and hacks. Users are expected to know their way around using (and especially debugging) Nix and Emacs Lisp before using this project.
+
+If you encounter any issues that make it unusable to you (or if you need support), please talk to us first in the [Matrix room](https://matrix.to/#/#doom-emacs:nixos.org) and if it's indeed a bug of Nix-Doom-Emacs, file it in the [issue tracker](https://github.com/nix-community/nix-doom-emacs/issues).
+
+If you find this documentation unclear or incomplete, please let us know as well.
+
+Here's the [FAQ](./faq.md)
+
+Nix-Doom-Emacs uses [`nix-straight.el`](https://github.com/nix-community/nix-straight.el) under the hood to install dependencies. It's a low level wrapper to add Nix integration over [`straight.el`](https://github.com/radian-software/straight.el), the declarative package manager used by Doom Emacs. 
+
+Before using Nix-Doom-Emacs, make sure to read [`nix-straight.el`'s README'](https://github.com/nix-community/nix-straight.el), and that you understand the consequences.
+
+# Getting Started
+
+To get started, we suggest these methods. They are ordered from most suggested to least suggested.
+In all of these methods, you'll need your Doom Emacs configuration. It should contain the following three files: 
+`config.el`, `init.el` and `packages.el`. If you don't already have an existing `doom-emacs` configuration, you can use the contents of `test/doom.d` as a template.
+
+The Doom configuration will be referred to as `./doom.d` in these snippets. You can name it whatever you like.
+
+## Flake + Home-Manager
+
+`File: flake.nix`
+```nix
+{
+  inputs = {
+    home-manager.url = "github:nix-community/home-manager";
+    nix-doom-emacs.url = "github:nix-community/nix-doom-emacs";
+  };
+
+  outputs = {
+    self,
+    nixpkgs,
+    lib,
+    home-manager,
+    nix-doom-emacs,
+    ...
+  }:
+    let
+      system      = "x86_64-linux";
+      specialArgs = { inherit inputs; };
```

Comment:
```suggestion
```

This is not being used here, and I think it doesn't make sense to confuse folks about `specialArgs`, getting at the risk that maybe will think this is essential for it to work.

Reply:
It's put there because it allows `home.nix` to get access to the `inputs` from the flake, which is something I think is very commonly needed, but I think I'll remove it since this isn't about getting the user an opinionated config

Reply:
Yeah, I know that passing `inputs` as `specialArgs` is common. I just don't want folks assuming that this is essential to get it working.